### PR TITLE
Bugfixes/gensched badreq

### PR DESCRIPTION
**Pull request by jonZlotnik**
I apologize for the ugly patch. I didn't wanna start messing with the other frontend code and break other functions and confuse code. If the people working on those sections wanna refactor to make the data flow nicer, go for it!

### CODE REVIEW

**Comment by jonZlotnik** [comment]
wait, I broke guest

**Comment by ferdousara-parvin** [comment]
I created a new user, added a course in the record and the sequence, added a semester and clicked on make my schedules. I got the following error when trying to load the course record, course sequence and semesters. I still went back to the user record page to check if my information was still there. Only the course added in the course sequence was visible. The info I added in the semesters section and in the record were gone.
image

File: skedge-backend/middlewares/decrypt-requests.js [hunk 1]
```diff
@@ -2,12 +2,12 @@ const decrypt = require("../web_api_utilities/rsa-encryption").decryptStringWith
 
 module.exports = function (req, res, next) {
     try {
-        let verbose_logging = false;
+        let verbose_logging = true;
```
Comment (felixlapierre1) [inline skedge-backend/middlewares/decrypt-requests.js]: I like this - being able to toggle the big logs is good. However I also think this should be set to false on the master branch, and only set to true temporarily when debugging.

That being said, don't start adding these ifs everywhere or the code will get messy. Ideally we would start using a logging framework like winston.

File: skedge-backend/middlewares/decrypt-requests.js [hunk 2]
```diff
@@ -20,6 +20,13 @@ module.exports = function (req, res, next) {
                 console.log("\n");
             }
         }
+        if(verbose_logging && req.method == "GET") 
+        {
+            if(verbose_logging) {
```
Comment (felixlapierre1) [inline skedge-backend/middlewares/decrypt-requests.js]: Remove useless if statement

File: skedge-backend/index.js [hunk 3]
```diff
@@ -207,6 +207,7 @@ app.post('/users/saveRecAndSeq', checkAuth, (req, res, next) => {
 // "If there’s nothing, don’t crash." --> That should be ensured by the database?
 app.post('/users/loadRecAndSeq', checkAuth, (req, res, next) => {
     let userId = req.body.authToken.userId;
+    console.log(userId);
```
Comment (felixlapierre1) [inline skedge-backend/index.js]: Remove console log

File: skedge-backend/middlewares/decrypt-requests.js (hunk 1 above)
Comment (jonZlotnik) [reply skedge-backend/middlewares/decrypt-requests.js]: yeah... a logging framework would be nice. Was really just for this module. Will update the var

File: skedge-backend/middlewares/decrypt-requests.js (hunk 2 above)
Comment (jonZlotnik) [reply skedge-backend/middlewares/decrypt-requests.js]: oh... I'm dumb

File: skedge-backend/index.js (hunk 3 above)
Comment (jonZlotnik) [reply skedge-backend/index.js]: fanks

**Pull request merged**